src/cli/Utilities.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+export class Utilities{
+    /**
+     * Check to ensure filename only contains valid characters
+     * @param {fileName} string - filename to be evaluated
+     * @returns {Promise<boolean>} - promise that resolves to true if filename contains valid characters
+     * @memberof Utilities
+     */
+    public static async isValidFileName(fileName: string): Promise<boolean> {
```

Comment:
It seems `async` is unnecessary here?

Reply:
@std4lqi thank you!! copy pasted the method signature from a different file and didn't tailor anything but the method name 😅